File: src/main/java/com/github/reallyliri/limonitprogressbarplugin/Res/Colors.java
package com.github.reallyliri.limonitprogressbarplugin.Res;

import com.intellij.ui.JBColor;
import org.jetbrains.annotations.NotNull;

import java.awt.*;
import java.util.List;

public class Colors {
    private static final Color ORANGE_LIGHT = new JBColor("ORANGE_LIGHT", new Color(218, 154, 88));
    private static final Color ORANGE_DARK = new JBColor("ORANGE_DARK", new Color(179, 99, 29));
    private static final Color BROWN = new JBColor("BROWN", new Color(70, 35, 13));

    @NotNull
    private static JBColor toJBColor(@NotNull Color color) {
        return new JBColor(color, color);
    }

    public static final List<Color> COLORS = List.of(
            ORANGE_LIGHT,
            ORANGE_DARK,
            BROWN,
            ORANGE_DARK,
            ORANGE_LIGHT
    );
}

File: src/main/kotlin/com/github/reallyliri/limonitprogressbarplugin/Colors.kt
package com.github.reallyliri.limonitprogressbarplugin

import java.awt.Color

interface Colors {
    companion object {
        private val RED: Color = Color(255, 0, 0)
        private val ORANGE: Color = Color(255, 154, 0)
        private val YELLOW: Color = Color(255, 255, 0)
        private val GREEN: Color = Color(25, 255, 0)
        private val BLUE: Color = Color(0, 150, 255)
        private val VIOLET: Color = Color(100, 30, 255)
        val RAINBOW_ARRAY: Array<Color> = arrayOf<Color>(RED, ORANGE, YELLOW, GREEN, BLUE, VIOLET)
    }
}